src/main/java/com/uber/cadence/internal/compatibility/proto/serviceclient/GrpcServiceStubs.java

```diff
@@ -53,7 +54,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-final class GrpcServiceStubs implements IGrpcServiceStubs {
+@VisibleForTesting
+public final class GrpcServiceStubs implements IGrpcServiceStubs {
```

Comment:
same as below, it doesn't need to public

Reply:
Done